src/service/viewer-impl.js

```diff
@@ -221,6 +221,10 @@ export class Viewer {
     this.performanceTracking_ = this.params_['csi'] === '1';
     dev.fine(TAG_, '- performanceTracking:', this.performanceTracking_);
 
+    this.shareTrackingIncomingFragment_ = this.params_['share-tracking'];
```

Comment:
Yes. The `getParam` method would be enough, but because of server side rendering, the fragment cannot be known when the URL is generated. For that reason we must switch to a method that asks the viewer